plugins/hls-tactics-plugin/test/golden/GoldenListFmap.hs.expected

```diff
@@ -1,3 +1,3 @@
 fmapList :: (a -> b) -> [a] -> [b]
 fmapList _ [] = []
-fmapList fab (a : l_a3) = fab a : fmapList fab l_a3
+fmapList fab (a : as') = fab a : fmapList fab as'
```

Comment:
This should be `as`, not `as'` :confused:

Reply:
Ahh, `as` is the name being given the `[a]` parameter introduced by the lambda, which immediately gets split away. Fixing this is going to require a more global view of names than we currently have, so it wont happen today.